src/frame/mod.rs

```diff
@@ -420,6 +444,7 @@ impl Exception {
             MemoryParityError => "Memory parity error",
             GatewayPathUnavailable => "Gateway path unavailable",
             GatewayTargetDevice => "Gateway target device failed to respond",
+            Custom(_) => "Other",
```

Comment:
```suggestion
            Custom(_) => "Custom",
```

Reply:
We don't know if `Custom` actually contains a value that equals one of the above, i.e. _other_. Let's simply return the name of the variant.